CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ The main work (all changes without a GitHub username in brackets in the below li
 ## __WORK IN PROGRESS__
 
 -   @matter/node
+    - Feature: Constraint and conformance expressions may now reference values by name in any owner of a constrained value
```

Comment:
This also adds new validations to existing 1.3 cases right? If yes we might add a "potentially breaking" to description because we could validate stuff that was not validated before which could break existing code. So just text adjustment here in changelog.